src/plugins/wb-data-json/demo/data-en.json

```diff
@@ -40,13 +40,33 @@
       "name": "Mr X",
       "city": "Gatineau",
       "somethingHTML": "<strong>Horay</strong> here rich HTML content",
-      "hobby": [ "Car", "Mechanic", "Race" ]
+      "hobby": [ "Car", "Mechanic", "Race" ],
+      "language": {
+        "en": {
+          "description": "description of Mr X",
```

Comment:
@duboisp Not a big deal, but you never use the "description" field in your example.

Reply:
Je manquais d'inspiration, on peut toujours le changer si tu as une proposition. :-)